Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -162,6 +170,14 @@ class PreplannedMapModel: ObservableObject, Identifiable {
         }
     }
     
+    /// Removes the downloaded preplanned map area from disk and reset status.
+    func removeDownloadedPreplannedMapArea() async {
```

Comment:
A test should be added for this method in `PreplannedMapModelTests` to verify that the status.